app/views/layouts/_footer.html.erb

```diff
@@ -1,20 +1,23 @@
 <footer class="no-print">
   <div class="row">
     <h1>Security Education Companion</h1>
-
-    <div class="logos">
+  <div class="row logos">
```

Comment:
The extra `.row` here adds a slight horizontal scroll to the page. Looks like we can do without it?

Reply:
fixed this!

Reply:
Thanks. The indentation is also off around here. Other than that this looks good to go.